bench/app.py

```diff
@@ -129,7 +129,7 @@ def _setup_details_from_git_url(self, url=None):
 
 	def __setup_details_from_git(self, url=None):
 		name = url if url else self.name
-		if name.startswith("git@") or name.startswith("ssh://"):
+		if "@" in name or name.startswith("ssh://"):
```

Comment:
This breaks clones with PAT which do have `@` in them and follow this format:

`git clone https://{USER}:{PAT}@github.com/{REPO}`


I'd suggest using `ssh://` if you need to use non-`git` user.

Reply:
Alternatively reorder conditions so HTTP clones have higher priority and both cases get handled.